Tests/Functional/Type/DefinitionTest.php

```diff
@@ -53,12 +53,7 @@ public function testDefinesAnObjectTypeWithDeprecatedField()
 
     private function isDeprecated($node)
```

Comment:
Maybe the whole function can be removed?
it looks easier to call `$field->isDeprecated()`

Reply:
Nice catch :+1:  thank you

Reply:
Cool, are you going to remove it? I'm looking forward to merge it! :)

Reply:
yes i'm going to remove it